Documentation/cluster-crd.md

```diff
@@ -80,11 +81,31 @@ A Placement configuration is specified (according to the kubernetes [PodSpec](ht
 - `podAntiAffinity`: kubernetes [PodAntiAffinity](https://kubernetes.io/docs/api-reference/v1.6/#podantiaffinity-v1-core)
 - `tolerations`: list of kubernetes [Toleration](https://kubernetes.io/docs/api-reference/v1.6/#toleration-v1-core)
 
+### Cluster-wide Resources Configuration Settings
+
+**WARNING** Before setting resource requests/limits, please take a look at the Ceph documentation for recommendations for each component: [Ceph - Hardware Recommendations](http://docs.ceph.com/docs/master/start/hardware-recommendations/).
```

Comment:
how about if we remove this warning? The same warning is below in the detail section, right?